src/native/_mutable.js

```diff
@@ -105,7 +105,7 @@ const makeMDataInfo = (mDataInfoObj) => {
   }
   return new MDataInfo({
     name: mDataInfoObj.name,
-    type_tag: mDataInfoObj.type_tag,
+    typeTag: mDataInfoObj.typeTag,
```

Comment:
I'm not sure if it really matters but perhaps is worthy keeping the prop name here as `type_tag` as it's supposed to match the native struct, and only call it `typeTag` on the json we expose only